test/src/sdk-harness/test_artifacts/tx_output_predicate/src/main.sw

```diff
@@ -0,0 +1,14 @@
+predicate;
+
+use std::outputs::{output_asset_id, output_asset_to};
+
+fn main(index: u64, asset_id: ContractId, to: b256) -> bool {
+
+    let tx_asset_id = output_asset_id(index);
+    let tx_to = output_asset_to(index);
+
+    assert(tx_asset_id.is_some() && tx_asset_id.unwrap() == asset_id);
+    assert(tx_to.is_some() && tx_to.unwrap() == to);
```

Comment:
Quality is important but keep in mind that this is a test so unwrapping on a `None` is a valid failure since the test should not be `None` in the 1st place

Reply:
If either result is `None` the assertion will fail and therefore fail the test, making it a valid failure